review_code_gemini.py

```diff
@@ -60,61 +60,53 @@ def analyze_code(parsed_diff: List[Dict[str, Any]], pr_details: PRDetails) -> Li
     """Analyzes the code changes using Gemini and generates review comments."""
     print("Starting analyze_code...")
     print(f"Number of files to analyze: {len(parsed_diff)}")
-    #print(f"Files content: {json.dumps(parsed_diff, indent=2)}")
     comments = []
-    print(f"Initial comments list: {comments}")
+    #print(f"Initial comments list: {comments}")
     
     for file_data in parsed_diff:
         file_path = file_data.get('path', '')
         print(f"\nProcessing file: {file_path}")
-        #print(f"File data: {json.dumps(file_data, indent=2)}")
 
         if not file_path or file_path == "/dev/null":
             continue
-        
-       # Fixed PatchedFile initialization
-        # patched_file = PatchedFile(
-        #     source_file=f"a/{file_path}",
-        #     target_file=f"b/{file_path}",
-        #     is_binary_file=False
-        # )
```

Comment:
The variable `gemini_model` should be initialized outside the function, probably as a global variable, to avoid re-initializing it with every function call.